Components/Toast/toast.html

```diff
@@ -0,0 +1,174 @@
+</head>
+<body>
+    <div class="container">
+        <header class="navbar">
+            <span id="sidenav"><i class="fas fa-bars fa-lg"></i></span>
+            <span class="logo">
+            <img src="../../assets/swan-breeze.svg" alt="logo" width="35px" height="35px" />
+            <!-- <img src="../assets/logo.svg" alt="logo" width="35px" height="35px" /> -->
+            </span>
+            <a href="../../index.html" class="brand-name h6">
+            Breeze.css
+            </a>
+            <div class="navbar-items">
+            <nav>
+                <a href="https://github.com/chanchal16/breeze.css" class="github-btn" target="_blank">
+                <i class="fab fa-github fa-2x"></i>
+                </a>
+            </nav>
+            </div>
+        </header>
+        <aside>
+            <div class="drawer" id="sidebar">
+              <div class="drawer-content">
+                <h2 class="text-md"><strong>Components</strong></h2>
+                <div style="margin: auto;">
+                  <ul>
+                    <li class="drawer-items">
+                      <a href="../Alert/alert.html">Alerts</a>
+                    </li>
+                    <li class="drawer-items ">
+                      <a href="../Avatar/avatar.html"> Avatars</a>
+                    </li>
+                    <li class="drawer-items">
+                      <a href="../Badge/badge.html">Badge</a>
+                    </li>
+                    <li class="drawer-items">
+                      <a href="../Button/button.html">Button</a>
+                    </li>
+                    <li class="drawer-items">
+                      <a href="../Cards/cards.html">Cards</a>
+                    </li>
+                    <li class="drawer-items">
+                      <a href="../ImageComponent/image.html">Image</a>
+                    </li>
+                    <li class="drawer-items">
+                      <a href="../Input/input.html">Inputs</a>
+                    </li>
+                    <li class="drawer-items">
+                        <a href="../List/list.html">List</a>
+                    </li>
+                    <li class="drawer-items">
+                        <a href="../Modal/modal.html">Modal</a>
+                    </li>
+                    <li class="drawer-items">
+                        <a href="../Navbar/navbars.html">Navbars</a>
+                    </li>
+                    <li class="drawer-items">
+                        <a href="../Ratings/ratings.html">Ratings</a>
+                    </li>
+                    <li class="drawer-items">
+                        <a href="../Slider/slider.html">Slider</a>
+                    </li>
+                    <li class="drawer-items selected">
+                        <a href="">Toast</a>
+                    </li>
+                    <li class="drawer-items">
+                      <a href="../Text Utilities/typography.html">Typography</a>
```

Comment:
just a suggestion: try avoiding naming file names with spaces in between.

Reply:
that's folder name